Formula/vulkan-loader.rb

```diff
@@ -0,0 +1,45 @@
+class VulkanLoader < Formula
+  desc "Vulkan ICD Loader"
+  homepage "https://github.com/KhronosGroup/Vulkan-Loader"
+  url "https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.220.tar.gz"
+  sha256 "fe5b65e5d88febb1220ae59de363ed8b2794f8c861d6d74efb14aecefd464559"
+  license "Apache-2.0"
+  head "https://github.com/KhronosGroup/Vulkan-Loader.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python@3.10" => :build
+  depends_on "vulkan-headers"
+
+  on_linux do
+    depends_on "libx11"
+    depends_on "libxcb"
+    depends_on "libxrandr" => :build
+    depends_on "wayland"
+  end
+
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].opt_prefix}",
+                    "-DFALLBACK_DATA_DIRS=#{HOMEBREW_PREFIX}/share:/usr/local/share:/usr/share",
```

Comment:
Hmmmm, it looks like in practice the strings we end up with by default are just `/etc` and `/etc/xdg`, which aren't too unreasonable; I could see maybe setting `CMAKE_INSTALL_FULL_SYSCONFDIR` to `#{HOMEBREW_PREFIX}/etc` (which would leave `/etc` still being searched as well), and `FALLBACK_CONFIG_DIRS` to `#{HOMEBREW_PREFIX}/etc/xdg:/etc/xdg`, but… is anyone actually going to stick local config files there (vs in `/etc[/xdg]`) anyway?

Reply:
Actually I think it looks like in practice we get a path like
```
$HOMEBREW_PREFIX/Cellar/vulkan-loader/x.y.z/etc
```
which isn't great for configuration files because it gets deleted by `brew upgrade vulkan-loader`.

> but… is anyone actually going to stick local config files there (vs in `/etc[/xdg]`) anyway?

Writing to `/etc` requires `sudo`, and we aim to minimise needing it when using Homebrew.

Reply:
Oh, hah, it's even sillier than that, actually: we get `/etc/opt/homebrew/Cellar/vulkan-loader/1.3.220`. Yeah, that's pretty obviously nonsensical, I'll try to write a patch for it.

Reply:
Yea that's nonsense (and actually is inconsistent with [CMake documentation](https://cmake.org/cmake/help/v3.24/module/GNUInstallDirs.html#special-cases)).

It should be enough to set `SYSCONF` and/or `FALLBACK_CONFIG_DIRS`, I think.